keymaps/intellij-idea-keymap.cson

```diff
@@ -249,6 +249,13 @@
   # Move line down
   'ctrl-shift-down': 'editor:move-line-down'
 
+  # Delete to previous word
+  'ctrl-backspace': 'editor:delete-to-previous-word-boundary'
+
+  # Delete to next word
+  'ctrl-delete': 'editor:delete-to-next-word-boundary'
+
+
```

Comment:
Could you maybe remove one empty line such that there is only one empty line to the next line with content?